utils/setMaterialMap.js

```diff
@@ -0,0 +1,7 @@
+export function setMaterialMap(material, map) {
```

Comment:
This helper is not ideal since it performs operations which are not always required when setting a map. If you just want to exchange an existing texture there is no need to set `needsUpdate` to `true`.

`three.js` is designed in a way that `needsUpdate` should be used on app level.